manim/config/default.cfg

```diff
@@ -174,7 +174,7 @@ log_message =
 log_path = dim
 log_width = -1
 log_height = -1
-log_timestamps = True
+log_timestamps = False
```

Comment:
Question: is this necessary? I kinda like being able to tell how long the rendering took.

Reply:
This is not necessary, no. But I thought it would be appreciated as timestamps in the rich formatted logs are very, very annoying and ugly. 

I can change it back if you think it should be so.

Reply:
I have no strong opinion about showing logs or not.

I do have a strong opinion about changing `default.cfg` without first reaching consensus, because this affects every single user.

Reply:
Alright. I’ll change this back